variables.tf

```diff
@@ -142,6 +142,12 @@ variable "log_multiline_pattern" {
   type        = string
 }
 
+variable "log_configuration_options_overrides" {
+  description = "Overrides for log configuration options"
+  default     = {}
+  type        = any
```

Comment:
```suggestion
  type        = map
```

Reply:
At least when declaring a i.e. `object({key1: string, key2: number})`, it will only pass on the declared ones, so if I sent in `var_name = { keyNotDeclared = "someValue" }` it wouldn't be included where it was used.

Reply:
So it might work with `map`. I will take a look.

Reply:
You need to specify everything inside `map({ ... })` :/

Reply:
There is a defined number of options available for the hard coded awslogs driver in the module. These are documented here:
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_awslogs.html#create_awslogs_logdriver_options

We are currently setting the `awslogs-group`, `awslogs-region` and `awslogs-stream-prefix` options.

And, as you can see, we have already provided a way to override the `awslogs-group` option. Maybe you can use the same method to override `awslogs-region` and `awslogs-stream-prefix` instead?

Can you provide an example of why you need this?

Reply:
I agree with everything you write. It is way more than I initially needed, so lets just close this PR and also #81.